manifest/android-test.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
are we keeping this file here or in the other repo?

Reply:
just a note, some of the versions are outdated if we end up releasing a build

Reply:
this is just to have an example in that repo